src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -15,7 +15,7 @@ import {
 } from "../../../../gatsby-plugin-theme-ui/components"
 
 const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
-  <section id="use-cases">
+  <section id="use-cases" style={{scrollMarginTop:"-3.5rem"}} >
```

Comment:
might want to do this with tailwind but not a huge deal right now.

Reply:
It didn't do well with the tailwind so I had to increase specificity with inline CSS.